rest/server_context.go

```diff
@@ -664,7 +664,7 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 			}
 
 			if dbcontext.RevsLimit < 100 {
```

Comment:
(Sorry, should have got this in in one go) - should we extract this as a constant, so it's not a magic number buried in the code? Could also then sub it into the warning message...

Reply:
Swapped this out to use the default revs limit allow conflict constant here